n3fit/src/n3fit/scripts/evolven3fit_new.py

```diff
@@ -0,0 +1,376 @@
+    )
+    eko_parser = subparsers.add_parser(
+        "produce_eko",
+        help="Produce the eko for the specified theory. The q_grid starts at the Q0 given by the theory but the last point is q_fin and its number of points can be specified by q_points. The x_grid starts at x_grid_ini and ends at 1.0 and contains the provided number of points. The eko will be dumped in the provided path.",
+    )
+    evolvefit_parser.add_argument(
+        "configuration_folder",
+        help="Path to the folder containing the (pre-DGLAP) fit result",
+    )
+    parser.add_argument(
+        "-q",
+        "--q_fin",
+        type=float,
+        default=None,
+        help="Final q-value of the evolution",
+    )
+    parser.add_argument(
+        "-p",
+        "--q_points",
+        type=int,
+        default=None,
+        help="Number of q points for the evolution",
+    )
+    parser.add_argument(
+        "-n",
+        "--n_cores",
+        type=int,
+        default=1,
+        help="Number of cores to be used",
+    )
+    evolvefit_parser.add_argument(
+        "-l",
+        "--load",
+        type=pathlib.Path,
+        default=None,
+        help="Path of the EKO to be loaded",
+    )
+    evolvefit_parser.add_argument(
+        "-d",
+        "--dump",
+        type=pathlib.Path,
+        default=None,
+        help="Path where the EKO is dumped (optional)",
+    )
+    evolvefit_parser.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        help="Force the evolution to be done even if it has already been done",
+    )
+    eko_parser.add_argument(
+        "theoryID", type=int, help="ID of the theory used to produce the eko"
+    )
+    eko_parser.add_argument(
+        "dump",
+        type=pathlib.Path,
+        help="Path where the EKO is dumped",
+    )
+    eko_parser.add_argument(
+        "-i",
+        "--x_grid_ini",
+        default=None,
+        type=float,
+        help="Starting point of the x-grid",
+    )
+    eko_parser.add_argument(
+        "-p",
+        "--x_grid_points",
+        default=None,
+        type=int,
+        help="Number of points of the x-grid",
+    )
+    args = parser.parse_args()
+    op_card_info = {
+        "ev_op_max_order": 10,
+        "ev_op_iterations": 1,
+        "n_integration_cores": args.n_cores,
+        "backward_inversion": "expanded",
+    }
+    t_card_info = {}
+    if args.actions == "evolve":
+        cli_evolven3fit_new(
+            args.configuration_folder,
+            args.q_fin,
+            args.q_points,
+            op_card_info,
+            t_card_info,
+            args.dump,
+            args.load,
+            args.force,
+        )
+    elif args.actions == "produce_eko":
+        stdout_log = logging.StreamHandler(sys.stdout)
+        stdout_log.setLevel(logging.INFO)
+        stdout_log.setFormatter(
+            logging.Formatter("%(asctime)s %(name)s/%(levelname)s: %(message)s")
+        )
```

Comment:
Since this is also the logging formatter used inside the package, you can consider to make it a constant in the package, and use that constant here (instead of keeping copy-pasting the value)